io_export_objex2/data_updater.py

```diff
@@ -13,6 +13,7 @@
 #  You should have received a copy of the GNU General Public License
 #  along with this objex2 addon.  If not, see <https://www.gnu.org/licenses/>.
 
+from cgitb import text
```

Comment:
thonk
probably IDE auto-import PepeLaugh

Reply:
Might be, I have no idea what that even is. I'll get rid of it.